Remove redirect in link to PATCH RFC

timothystone opened the pull request:
Fixes #1588 

In the FAQ, Where's PUT?, the link under PATCH points to an insecure
location, e.g., http://tools.ietf.org/html/rfc5789 and results in
two (2) redirects to the correct location.

The correct URL, after redirection, is
https://datatracker.ietf.org/doc/html/rfc5789

Moreover, this can sometimes result in a 404 from tools.ietf.org
(which appears to be intermittent, possibly due to a server
misconfiguration; the personal website for Henrik Levkowetz, and
member of the IETF Tools Team, appeared at the root several times
before starting to redirect, see response in issue 🤔 ).

timothystone-knsl committed:
Remove redirect in link to PATCH RFC

Fixes #1588

In the FAQ, Where's PUT?, the link under PATCH points to an insecure
location, e.g., http://tools.ietf.org/html/rfc5789 and results in
two (2) redirects to the correct location.

The correct URL, after redirection, is
https://datatracker.ietf.org/doc/html/rfc5789

Moreover, this can sometimes result in a 404 from tools.ietf.org
(which appears to be intermittent, possibly due to a server
misconfiguration; the personal website for Henrik Levkowetz, and
member of the IETF Tools Team, appeared at the root several times
before starting to redirect, see response in issue 🤔 ).

jelhan commented on faq/index.md:
```diff
@@ -62,15 +62,15 @@ non-standard actions they support. Feel free to
   target resource will result in an equivalent representation being sent…”
 
-The correct method for partial updates, therefore, is [PATCH](http://tools.ietf.org/html/rfc5789),
+The correct method for partial updates, therefore, is [PATCH](https://datatracker.ietf.org/doc/html/rfc5789).
```
I guess the comma was replaced by a dot by accident.

```suggestion
The correct method for partial updates, therefore, is [PATCH](https://datatracker.ietf.org/doc/html/rfc5789),
```

timothystone replied:
D'oh. Yes. Thanks @jelhan. Committed the suggestion.

timothystone committed:
Correct the punctuation error

`s/\./,` replace the full stop (`.`) with a comma (`,`) that snuck through proofreading.

Co-authored-by: Jeldrik Hanschke <jelhan@users.noreply.github.com>

dgeb commented:
Thanks @timothystone and @jelhan! Appreciate the corrections.
